TicketsDemo.XML/XMLWorker.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+using TicketsDemo.Data.Entities;
+
+namespace TicketsDemo.XML
+{
+    class XMLWorker
+    {
+       
+        XMLSettingsService xml_set = new XMLSettingsService();
```

Comment:
Following C# common code style conventions, variables to be defined in camelCase, not bla_bla_case.